src/portal/src/i18n/lang/fr-fr-lang.json

```diff
@@ -80,26 +80,26 @@
         "PASSWORD": "Le mot de passe doit être composé de 8 à 128 caractères avec au moins 1 majuscule, 1 minuscule et 1 chiffre.",
         "CONFIRM_PWD": "Les mots de passe ne correspondent pas.",
         "SIGN_IN_USERNAME": "Le nom d'utilisateur est obligatoire.",
-        "SIGN_IN_PWD": "Le mot de passe obligatoire.",
+        "SIGN_IN_PWD": "Le mot de passe est obligatoire.",
         "SIGN_UP_MAIL": "L'email n'est utilisé que pour réinitialiser votre mot de passe.",
         "SIGN_UP_REAL_NAME": "Prénom et Nom",
         "ITEM_REQUIRED": "Le champ est obligatoire.",
-        "SCOPE_REQUIRED": "Le champ est obligatoire et doit  être au format scope.",
+        "SCOPE_REQUIRED": "Le champ est obligatoire et doit être au format scope.",
         "NUMBER_REQUIRED": "Le champ est obligatoire et doit être numérique.",
         "PORT_REQUIRED": "Le champ est obligatoire et doit être un numéro de port valide.",
         "CRON_REQUIRED": "Le champ est obligatoire et doit être au format cron.",
         "EMAIL_EXISTING": "L'adresse e-mail existe déjà.",
         "USER_EXISTING": "Le nom d'utilisateur est déjà utilisé.",
         "NONEMPTY": "Ne peut pas être vide",
         "REPO_TOOLTIP": "Les utilisateurs ne peuvent effectuer aucune opération sur les images dans ce mode.",
-        "ENDPOINT_FORMAT": "Le point de terminaison doit commencer par HTTP:// or HTTPS://.",
-        "OIDC_ENDPOIT_FORMAT": "Le point de terminaison doit commencer par HTTPS://.",
+        "ENDPOINT_FORMAT": "Le endpoint doit commencer par HTTP:// or HTTPS://.",
```

Comment:
Because in French it doesn't sound well, did you already head someone talking about "point de terminaison" ?
WDYT @TheoGthr ?

PS: I agree with "l'endpoint", I thought I fixed it everywhere but no